tests/acceptance/pageObjects/loginPage.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
   elements: {
     body: 'body',
     authenticateButton: {
-      selector: '#authenticate'
+      selector: '//div/button/span[@class="MuiButton-label"]',
```

Comment:
that is a very generic classname, please use specific id or the text of the button with`Log in`